spec/board_spec.rb

```diff
@@ -37,20 +37,51 @@
   end
 
   describe '#valid_placement?' do
-    
-    it 'checks if the placement is equal to length of ship' do
+
+    it 'returns false if placement length is equal to length of ship' do
       board = Board.new
       cruiser = Ship.new("Cruiser", 3)
 
       expect(board.valid_placement?(cruiser, ["A1", "A2"])).to be false
     end
 
-    it 'checks if coordinates are consecutive' do
+    it 'returns true if placement length is equal to length of ship' do
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
+
+      expect(board.valid_placement?(cruiser, ["A1", "A2", "A3"])).to be true
+    end
+
+    it 'returns false if coordinates are not consecutive horizontally' do
       board = Board.new
       cruiser = Ship.new("Cruiser", 3)
 
       expect(board.valid_placement?(cruiser, ["A1", "A2", "A4"])).to be false
     end
-  end
 
+    it 'returns true if coordinates are consecutive horizontally' do
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
+
+      expect(board.valid_placement?(cruiser, ["A1", "A2", "A3"])).to be true
```

Comment:
These tests look great! Can we add both instances of a submarine and a cruiser on each test to double check?

Reply:
Yes! That seems like a good safeguard